Cargo.toml

```diff
@@ -81,7 +81,7 @@ fuel-core-wasm-executor = { version = "0.27.0", path = "./crates/services/upgrad
 fuel-core-xtask = { version = "0.0.0", path = "./xtask" }
 
 # Fuel dependencies
-fuel-vm-private = { version = "0.51.0", package = "fuel-vm", default-features = false }
+fuel-vm-private = { version = "0.52.0", package = "fuel-vm", default-features = false }
```

Comment:
The changelog says your updating to `0.51.0`, not `0.52.0`.

Reply:
```
[#1888](https://github.com/FuelLabs/fuel-core/pull/1888): Upgraded `fuel-vm` to `0.51.0`. See [release](https://github.com/FuelLabs/fuel-vm/releases/tag/v0.51.0) for more information.
```

Reply:
The changelog related for `0.51.0` is from another PR, I just grouped everything together=)

Reply:
seems like we are being inconsistent about when we note these things, but I don't know that it matters